rclcpp/src/rclcpp/subscription_base.cpp

```diff
@@ -244,6 +244,7 @@ SubscriptionBase::take_serialized(
     &message_out.get_rcl_serialized_message(),
     &message_info_out.get_rmw_message_info(),
     nullptr);
+  TRACETOOLS_TRACEPOINT(rclcpp_take, static_cast<const void *>(&message_out.get_rcl_serialized_message()));
```

Comment:
This line is unfortunately too long. It should be <= 100 characters long. See: https://build.ros2.org/job/Rpr__rclcpp__ubuntu_noble_amd64/7/testReport/junit/(root)/projectroot/cpplint/

Reply:
@christophebedard 
Sorry, Fixed in https://github.com/ros2/rclcpp/pull/2448/commits/7050982430255fdca7c8530fe5bbb115ff2248f4.